tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -18,16 +19,36 @@
 )
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_osd_pods
 from ocs_ci.ocs.resources import storage_cluster
 from ocs_ci.utility.utils import ceph_health_check
 from ocs_ci.ocs.cluster import CephCluster
 from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
 from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
 
 
+logger = logging.getLogger(__name__)
+
+
 def add_capacity_test():
     osd_size = storage_cluster.get_osd_size()
+    existing_osd_pods = get_osd_pods()
+    existing_osd_pod_names = [pod.name for pod in existing_osd_pods]
     result = storage_cluster.add_capacity(osd_size)
+    osd_pods_post_expansion = get_osd_pods()
+    osd_pod_names_post_expansion = [pod.name for pod in osd_pods_post_expansion]
```

Comment:
add_capacity()  seems like is updating the device count and returning it.
After add-capacity are we waiting for the new osd pods to reach running state? If not there are chances that osd_pod_names_post_expansion list  contains only the osd's before expansion.

Reply:
the intention is the examine the already existing OSDs, not the newly created ones. Hence, it doesn't matter if the new OSDs were created already or not